index.html

```diff
@@ -0,0 +1,1942 @@
+					  </h4>
+					</article>
+					<div class="latest-news col-md-4">
+					  <div class="content">
+						<div class="head-story">
+						  <ul class="hidpi-subsection">
+							<li class="d-flex">
+							  <div class="image">
+								<a href="#">
+								  <img class="img-fluid" src="images/HiDPI_53.jpg"
+									alt="UFC 246: Conor McGregor Returns to Take on Donald Cerrone">
+								</a>
+							  </div>
+							  <div class="info">
+								<a href="#">UFC 246: Conor McGregor Returns to Take on Donald Cerrone</a>
+							  </div>
+							</li>
+							<li class="d-flex">
+							  <div class="image">
+								<a href="#">
+								  <img class="img-fluid" src="images/HiDPI_54.jpg"
+									alt="Titans vs. Chiefs Odds: Betting Lines and Trends for AFC Championship Game">
+								</a>
+							  </div>
+							  <div class="info">
+								<a href="#">Titans vs. Chiefs Odds: Betting Lines and Trends for AFC Championship Game</a>
+							  </div>
+							</li>
+							<li class="d-flex">
+							  <div class="image">
+								<a href="#">
+								  <img class="img-fluid" src="images/HiDPI_55.jpg"
+									alt="All You Need to Know about LSU's Title Parade on Saturday">
+								</a>
+							  </div>
+							  <div class="info">
+								<a href="#">All You Need to Know about LSU's Title Parade on Saturday</a>
+							  </div>
+							</li>
+						  </ul>
+						</div>
+					  </div>
+					</div>
+				  </div>
+				</div>
+			  </div>
+			</div>
+		  </div>
+	
+		  <!--  End HiDPI screen sotories -->
+	
+		  <div class="subscribe-block">
+			<div class="row">
+			  <div class="column1 subscribe col-lg-7 col-xl-8 d-flex">
+				<div class="head-story-red">
+				  <p>subscribe</p>
+				</div>
+				<div class="subscribe-wrap content">
+				  <div class="">
+					<div class="placement-wrap">
+					  <div class="image-wrap">
+						<div class="last-week">
+						  <img class="" src="images/january-03-17-2020-cover.jpg" alt="">
+						</div>
+						<div class="next-week">
+						  <img class="" src="images/january-24-31-2020-cover.jpg" alt="">
+						</div>
+					  </div>
+					  <div class="info">
+						<h3>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h3>
+						<p>All Access + Weekly Delivery.
+						  <br>
+						  Choose A Membership That's Perfect for You!
+						</p>
+						<a href="#" class="red-button">subscribe</a>
+					  </div>
+					</div>
+				  </div>
+				</div>
+			  </div>
+			  <div class="column2 newsletter col-lg-5 col-xl-4 d-flex">
+				<div class="head-story-red">
+				  <p>newsletter</p>
+				</div>
+				<div class="c2-subscribe-wrap d-flex">
+	
+				  <div class="newsletter-image"></div>
+				  <div class="c2-info">
+					<h3>The Starting 5</h3>
+					<p>Take back your inbox.
+					  Sign up for our newsletter and start making sense of the world around you.
+					</p>
+					<a href="#" class="red-button">sign up</a>
+				  </div>
+	
+		
```

Comment:
- [ ] This newsletter div is not displaying the image.

Reply:
The img folder was removed to images, which caused the error,
The img display fixed